### ISSUE TITLE
Create one place for community projects around OpenSearch

### ISSUE THREAD

**Issue by dblock** [opensearch-project/project-website#678]
**Is your feature request related to a problem? Please describe.**

It's hard to find 1) what projects are built around or connect to OpenSearch, and 2) what projects need help connecting to OpenSearch.

**Describe the solution you'd like**

I'd like something similar to https://api.slack.com/community where all the community projects that connect to or are built around OpenSearch are catalogued (including links to PRs where someone added support the first time), along with a way to ask for help to fix/extend an existing project (link to a GitHub issue that asks to support OpenSearch).

**Additional context**

Coming from https://github.com/spring-projects/spring-data-elasticsearch/issues/1918.

**Comment by jkowall** [comment]
I think it actually belongs here potentially, but this page also needs to be linked better: https://opensearch.org/connect.html

**Comment by elfisher** [comment]
There is the [community project page](https://opensearch.org/community_projects) on the website that aims to help with problem 1. Definitely opportunities to improve that page.

Do we also want to highlight projects that need help on that page? We already use the forums/GitHub to discuss issues and plan new work. We could create a place to track new projects in GitHub and highlight which ones want help.

**Comment by stockholmux** [comment]
I'm with @elfisher here. The community projects page is the way to go. 

I think adding some filtering (and maybe more fields) would help achieve what @dblock is looking for without wheel recreation.

**Comment by krisfreedain** [comment]
I would agree with @stockholmux & @elfisher - continuing to build out the existing Community Projects page with filtering would be the way to accomplish this.

**Comment by dblock** [comment]
The existing page is great as an advertisement for these higher level applications. For a developer audience I think we need a dedicated page sorted by language/platform.

**Comment by elfisher** [comment]
> The existing page is great as an advertisement for these higher level applications. For a developer audience I think we need a dedicated page sorted by language/platform. Some projects like "a python client" doesn't deserve a page, just a 1-liner, too. Adding a whole page seems like a very high bar.

We could improve that page to highlight those characteristics and create a search/filter function on those characteristics.

**Comment by stockholmux** [comment]
Revisiting this issue. I'm glad to work with anyone who can come up with a reasonable taxonomy to support filtering on the existing community projects page.
